example/spec/parallel/createReconciliationReport/IngestGranule.MYD13Q1_006.input.payload.json

```diff
@@ -31,11 +31,5 @@
         }
       ]
     }
-  ],
-  "pdr": {
```

Comment:
We should consider a generic CL note that we removed PDR eval from <list of tests>